cla_backend/apps/legalaid/utils/csvupload/validators.py

```diff
@@ -441,11 +532,41 @@ def _validate_determination_dvca_is_family(self, cleaned_data, category):
         if determination == u"DVCA" and category != u"family":
             raise serializers.ValidationError("Category (%s) must be Family if Determination is DVCA" % category)
 
+    def _validate_fixed_fee_amount_present(self, cleaned_data):
+        fixed_fee_code = cleaned_data.get("Fixed Fee Code")
+        if fixed_fee_code in ["DF", "LF", "HF"]:
```

Comment:
Shouldn't this be using `contract_2018_fixed_fee_codes` variable or are these different codes

Reply:
Yep, thanks 👍 Backported fix from the continuation branch ✅